storey/utils.py

```diff
@@ -42,6 +43,13 @@ def get_one_unit_of_duration(string_time):
     return multiplier
 
 
+def get_window_start_time_by_time(timestamp, window_millis):
+    midnight = datetime.fromtimestamp(int(timestamp/1000)).replace(hour=0, minute=0, second=0)
```

Comment:
Looks to me like this is where the time zone fails CI. The timestamp would be epoch (UTC), but midnight will depend on the local time zone.

Also, what's special about midnight? Why should we use it to determine the window start time, rather than a generic window aligned to epoch?